src/components/Recipes.js

```diff
@@ -0,0 +1,33 @@
+import '../assets/css/Recipes.css';
+import RecipeCard from "./RecipeCard";
+import getRecipeId from '../utils/getRecipeId';
+
+function Recipes({recipes, favRecipes, isFavorites, setRecipes, setFavRecipes}) {
+
+    const indexOfFavRecipe = function(recipe) {
+        return favRecipes.findIndex(r => getRecipeId(r) === getRecipeId(recipe));
+    } 
+
+    const toggleFav = function(recipe) {
+        const newFavRecipes = [...favRecipes];
+        const indexOfRecipe = indexOfFavRecipe(recipe);
+        if (indexOfRecipe === -1) {
+            newFavRecipes.push(recipe);
+        } else {
+            newFavRecipes.splice(indexOfRecipe, 1);
+        }
+        setFavRecipes(newFavRecipes);
+    }
+
+    return (<> 
+        <div className="recipe-list-header">{isFavorites ? "Favorite recipes" : "Found recipes"}</div>
+        <div className="recipe-list-container">
+            {recipes.map((recipe, index) => <RecipeCard recipe={recipe} 
```

Comment:
In this div i got a warning about keys:
Warning: Each child in a list should have a unique "key" prop, on the other side the key prop is defined

Reply:
Added this to the known issues